MilitaryManager.Units.API/Controllers/WeatherForecastController.cs

```diff
@@ -24,16 +25,20 @@ public WeatherForecastController(ILogger<WeatherForecastController> logger)
         }
 
         [HttpGet]
-        public IEnumerable<WeatherForecast> Get()
+        [Authorize]
+        //public IEnumerable<WeatherForecast> Get()
```

Comment:
What a reason of modify current controller? 

Please do not leave any commented code